package exceptionhandling;

public class Demo2 {
	/*public static void main(String[] args) {
		
		
		try {
			Exception e=new Exception();
			throw e;
		} catch (Exception e1) {
			// TODO Auto-generated catch block
			System.out.println("caught exception");
		}
	}*/
	public static void main(String[] args) throws Exception {
		Exception e=new Exception();
		throw e;
	}

}
